src/qibolab/sequence.py

```diff
@@ -73,14 +95,35 @@ def concatenate(self, other: "PulseSequence") -> None:
             - necessary delays to synchronize channels
             - ``other``
         """
-        durations = {ch: self.channel_duration(ch) for ch in other.channels}
-        max_duration = max(durations.values(), default=0.0)
-        for ch, duration in durations.items():
-            delay = max_duration - duration
-            if delay > 0:
-                self.append((ch, Delay(duration=delay)))
+        _synchronize(self, other.channels)
         self.extend(other)
 
+    def align(self, channels: list[ChannelId]) -> Align:
+        """Introduce align commands to the sequence."""
+        align = Align()
+        for channel in channels:
+            self.append((channel, align))
+        return align
+
+    def align_to_delays(self) -> "PulseSequence":
```

Comment:
Oh, you're definitely right. I could make it `.trimmed`.

But yes, it seems neat also as a function, because we know it's doing something (while from a property you generally expect to do little). Problem is that "something" is ill-defined (in a sense, each property does *something*, otherwise it would be an attribute), that's why the criterion above...